docs/usecase.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+left to right direction
+skinparam packageStyle rectangle
+actor customer
+actor receptionist
+rectangle "Camping reservation system" {
+  customer <|-- (Generate invoice based on reservation details)
+  customer -- (Search availability)
+  customer -- (Book a camping)
+  (Book a camping) .> (Electricity) : <<include>>
+  (Manage reservations) -- receptionist
```

Comment:
A `manage reservations` szükségtelen, és nem is include kapcsolatban áll a többivel.
Szerintem kihagyható és közvetlenül a `book a camping` és a `cancel reservation` a recepcióshoz húzható.